README.md

```diff
@@ -15,14 +15,14 @@ You'll need a Temasys Account and an App key to use this. [Register here to get
 | -------------- | ---------- | ----------- | --------- | ---------- | ---------- | ---- | ----- |
 | **Platforms:** | Win/Mac/Ubuntu/Android | Win/Mac/Ubuntu/Android | Win/Mac/Ubuntu/Android | Win/Mac | Win | Mac | Win | iOS<br>(`9.x` and below) |
```

Comment:
@ncurrier actually this was based off the AdapterJS support for bowser supports which is 9.x and below:
https://github.com/Temasys/AdapterJS#supported-browsers--platform-functionality

Bowser browser doesn't work with 10.x and above.